testsuite/forge-cli/src/main.rs

```diff
@@ -1419,6 +1421,28 @@ fn large_db_test(
         )
 }
 
+fn quorum_store_reconfig_enable_test(forge_config: ForgeConfig<'static>) -> ForgeConfig<'static> {
+    forge_config
+        .with_initial_validator_count(NonZeroUsize::new(20).unwrap())
+        .with_initial_fullnode_count(20)
+        .with_network_tests(vec![&QuorumStoreOnChainEnableTest {}])
+        .with_success_criteria(
+            SuccessCriteria::new(5000)
+                .add_no_restarts()
+                .add_wait_for_catchup_s(240)
+                .add_system_metrics_threshold(SystemMetricsThreshold::new(
+                    // Check that we don't use more than 12 CPU cores for 30% of the time.
+                    MetricsThreshold::new(12, 30),
+                    // Check that we don't use more than 10 GB of memory for 30% of the time.
+                    MetricsThreshold::new(10 * 1024 * 1024 * 1024, 30),
+                ))
+                .add_chain_progress(StateProgressThreshold {
+                    max_no_progress_secs: 10.0,
+                    max_round_gap: 4,
+                }),
```

Comment:
Magic numbers!  I just suggest for future readability, anything that's supposed to be the same / be a unit, it would be great to change it into a constant.  E.g. `NUM_NODES = 20` or `GB = 1024 * 1024 * 1024`

Reply:
I will try to get to this in a separate PR. There are too many tests that this would affect.